OoyalaSkinSampleApp/app/src/main/AndroidManifest.xml

```diff
@@ -65,6 +65,11 @@
                   android:configChanges="keyboardHidden|orientation|screenSize">
         </activity>
 
+        <activity android:name=".players.CustomConfiguredSkinPlayerActivity"
```

Comment:
Can you change the name to "ResizeSkinPlayerActivity", or "ResizablePlayerActivity" or something similar?  The CustomConfigured is something specifically for IMA and Freewheel